metabase/metabase
# Update convert query modal to match the preview modal

Epic https://github.com/metabase/metabase/issues/26837

Changes:
- Extracts reusable code to `NativeQueryModal`
- Makes both `PreviewQueryModal` and `ConvertQueryModal` use it

How to test:
- New -> Question -> Orders
- Click on the sql button in the header
- You should see the modal with a sql query preview
- It should be possible to convert the question to sql

<img width="1522" alt="Screenshot 2022-12-08 at 15 21 18" src="https://user-images.githubusercontent.com/8542534/206457000-67cc0954-e96c-4885-86d5-77452bd68add.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27072)
